tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -260,6 +281,33 @@ private string GetImage(string imageType, string version, string osVariant)
             return image;
         }
 
+        private static string GetRegistryName(string repo, string tag)
```

Comment:
Technically, this name isn't correct because in the case of the staging location, it is a registry plus a repo prefix.

Reply:
Renamed to `GetRegistryNameWithRepoPrefix`.